configure

```diff
@@ -592,11 +592,10 @@ case "$menhir_ver" in
   20[0-9][0-9][0-9][0-9][0-9][0-9])
       if test "$menhir_ver" -ge $MENHIR_REQUIRED; then
           echo "version $menhir_ver -- good!"
-          menhir_dir=$(ocamlfind query menhirLib 2>/dev/null \
-                           | tr -d '\r' | tr '\\' '/') || \
-          menhir_dir=$(menhir --suggest-menhirLib \
-                           | tr -d '\r' | tr '\\' '/') || \
+          menhir_dir=$(ocamlfind query menhirLib 2>/dev/null) || \
+          menhir_dir=$(menhir --suggest-menhirLib) || \
```

Comment:
One could also redirect stderr for the second command.

Reply:
I thought about it, but the two commands are a bit different: it is normal that `ocamlfind query menhirLib` fails (if Menhir wasn't installed through ocamlfind), so we don't want to show the error message; it is not normal that `menhir --suggest-menhirLib` fails, that's indicative of a problem in Menhir itself, so maybe we want to see the full error message to help track down the problem.

Reply:
Makes sense.